core/network/src/main/kotlin/com/google/samples/apps/nowinandroid/core/network/demo/DemoNiaNetworkDataSource.kt

```diff
@@ -57,6 +76,24 @@ class DemoNiaNetworkDataSource @Inject constructor(
     override suspend fun getNewsResourceChangeList(after: Int?): List<NetworkChangeList> =
         getNewsResources().mapToChangeList(NetworkNewsResource::id)
 
+    /**
+     * Convert [InputStream] to [String].
+     */
+    private suspend fun convertStreamToString(inputStream: InputStream): String = withContext(
```

Comment:
We don't need to reinvent the wheel 😛 
Kotlin already has extensions to `readText()` from input streams.

Reply:
Thanks, I'll test it.

Reply:
readText is working.
https://github.com/Jaehwa-Noh/nowinandroid/actions/runs/12426949721/job/34695993732